src/Sylius/Bundle/AdminBundle/Templating/Helper/VariantPriceHelper.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AdminBundle\Templating\Helper;
+
+use Sylius\Bundle\MoneyBundle\Formatter\MoneyFormatterInterface;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Sylius\Component\Currency\Model\CurrencyInterface;
+use Symfony\Component\Templating\Helper\Helper;
+
+class VariantPriceHelper extends Helper
```

Comment:
Could be final

Reply:
i can't do it because behats don't allow this class final :(
![Screenshot 2019-04-26 at 12 04 24](https://user-images.githubusercontent.com/29897151/56800500-73570800-681b-11e9-919f-385bd641581b.png)

Reply:
😢